policyengine_it/variables/gov/agenzia_delle_entrate/credits/low_income/low_income_credit.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_it.model_api import *
+
+from policyengine_it.variables.household.person.employment_category import (
```

Comment:
dont need to import here as we will use it in the var

Reply:
Fixed in a previous PR